### PR TITLE
NMS-9416: Add eap-ttls protocol to radius poller

### PR DESCRIPTION
**Pull request by jm-k**

### CODE REVIEW

**Comment by j-white** [comment]
There are some conflicts here since we merged https://github.com/OpenNMS/opennms/pull/1534

File: protocols/radius/src/main/java/org/opennms/protocols/radius/monitor/RadiusAuthMonitor.java [hunk 1]
```diff
@@ -168,6 +187,10 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
         String secret = ParameterMap.getKeyedString(parameters, "secret", DEFAULT_SECRET);
         String authType = ParameterMap.getKeyedString(parameters, "authtype", DEFAULT_AUTH_TYPE);
         String nasid = ParameterMap.getKeyedString(parameters, "nasid", DEFAULT_NASID);
+        String innerProtocol = ParameterMap.getKeyedString(parameters, "inner-protocol", DEFAULT_TTLS_INNER_AUTH_TYPE);
```
Comment (j-white) [inline protocols/radius/src/main/java/org/opennms/protocols/radius/monitor/RadiusAuthMonitor.java]: Please document the additional monitor parameters in `opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/RadiusAuthMonitor.adoc`

**Comment by soleger** [comment]
I'll rebase this against latest develop and see if the tests run properly.

**Comment by soleger** [comment]
I'm going to close this PR in favor of #1629.

**Pull request closed without merging**